docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -545,32 +577,59 @@ Go::
 
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
 
-func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+import (
+	"context"
+	"fmt"
+	"log"
 
-  cb.Cluster().SetName("hello-world")
+	"github.com/hazelcast/hazelcast-go-client"
+)
 
-  client, err := hazelcast.StartNewClientWithConfig(cb) 
+func main() {
+	ShutdownProperly := func(ctx context.Context, hz *hazelcast.Client, err error) {
+		errSd := hz.Shutdown(ctx)
+		switch {
+		case err != nil && errSd == nil:
+			log.Fatal(fmt.Errorf("%v", err))
+		case err != nil && errSd != nil:
+			log.Fatal(fmt.Errorf("%v; %v", err, errSd))
+		}
+	}
 
-  <1>
-  ctx := context.TODO()
-  myMap, err := client.GetMap(ctx, "my-distributed-map")
-  if err != nil {
-    // handle the error
-  }
-  entries, err := myMap.GetEntrySet(ctx)
-  if err != nil {
-    // handle the error
-  }
-  for key, value := range entries {
-    fmt.Println(key, value)
-  }
+	cb := hazelcast.NewConfig()
+	cb.ClientName = "hello-world"
+
+	ctx := context.TODO()
 
-  hz.Shutdown() <2>
+	hz, err := hazelcast.StartNewClientWithConfig(ctx, cb)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+  <1>
+	mp, err := hz.GetMap(ctx, "my-distributed-map")
+	if err != nil {
+		ShutdownProperly(ctx, hz, err)
+	}
+
+	entries, err := mp.GetEntrySet(ctx)
+	if err != nil {
+		ShutdownProperly(ctx, hz, err)
+	}
+
+	for key, value := range entries {
```

Comment:
Variable "key" refers to the index on the "value" here. I think it would be great if we can use the actual key of the entry.
```suggestion
	for _, entry := range entries {
		fmt.Printf("key:%s, value:%s\n", entry.Key, entry.Value)
	}
```